src/schemas/vis-state-schema.js

```diff
@@ -648,7 +648,11 @@ export const filterPropsV1 = {
       name: null,
       type: null
     }
-  })
+  }),
+  fixedDomain: null,
```

Comment:
fixedDomain is a property generated from `getDefaultFilter` we don't need to save it to the config

Reply:
ping